modules/nodejs-agent/test/plugins/http/expected.data.yml

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+segmentItems:
+  - serviceName: test-http
+    segmentSize: 1
+    segments:
+      - segmentId: 1.248349.185910205007481
+        spans:
```

Comment:
I think we missed the ref check here. And e2e should include the client/server spans

Reply:
> I think we missed the ref check here. And e2e should include the client/server spans

It’s plugin tests, not e2e

Reply:
Yes, this is not e2e test, I mean, the client to server rpc. We should check the header propagation and ref data.